cmd/system-probe/config/adjust_usm.go

```diff
@@ -52,8 +52,9 @@ func adjustUSM(cfg config.Config) {
 
 	if cfg.GetBool(smNS("process_service_inference", "enabled")) &&
```

Comment:
Just an FYI, this flag is still under `service_monitoring_config`
does it make sense to you to use it in NPM?
Should we consider deprecating it (with backward compatibility) in favor of a new shared value?

Reply:
I was considering deprecating it and changing it to a shared value... where would you think the value could most appropriately be placed?

Reply:
I'd say system_probe_config, but not sure if it is the correct location anymore as there are more products than just usm/npm in SP.